### DISCUSSION TITLE
How do you like the new changelog?

### DISCUSSION THREAD

**Discussion by vojtapolasek** [ComplianceAsCode/content#7347] [Announcements]
Dear community,
with the latest release (0.1.57), we have changed the way in which we generate the changelog. Also its structure has changed.
We use labels to describe changes within the PR. At the time of release, these labels are used to generate the changelog.
In the past, we did this process manually.
Currently, there are following sections in the changelog
- important highlights
- new rules and profiles
- updated rules and profiles
- changes in remediations
- changes in checks
- changes in the infrastructure
- changes in the test suite
- documentation

As this process was stabilized towards the end of the release cycle, the changelog does not contain all PRs, but I think it demonstrates the idea well enough.
What do you think about this approach? Suggestions, comments?
thank you.

**Comment by marcusburghardt** [comment]
For register, the used labels are defined in this file: https://github.com/ComplianceAsCode/content/blob/master/.github/workflows/release-changelog.json
